src/main/java/com/daitangroup/api/model/Person.java

```diff
@@ -1,11 +1,11 @@
 package com.daitangroup.api.model;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.Table;
+
+import org.codehaus.jackson.annotate.JsonProperty;
```

Comment:
Why are you using the older Jackson API?

Reply:
The newer is the fasterxml?

Reply:
Yes.